v_0_1_0/worker_template.go

```diff
@@ -168,13 +168,23 @@ coreos:
     command: stop
     mask: true
   - name: locksmithd.service
+    enable: false
```

Comment:
Looks like this change was not part of the closed PR. Is this on purpose?

Reply:
yes